### ISSUE TITLE
[BUG]: You need to provide at least one not-nullable parameter

### ISSUE THREAD

**Issue by joperator** [microsoft/azure-pipelines-agent#4242]
### What happened?

I updated a self-hosted Azure Pipelines Agent running on Arch Linux from version 3.214.0 to version 3.218.0. After that, all tasks running on this agent failed with `##[error]You need to provide at least one not-nullable parameter`. I found a workaround by adding `VERSION_ID="5.19.3"` to _/etc/os-release_. However, I expect the new agent version to work without this workaround.

### Versions

Azure Pipelines Agent version 3.218.0 / Arch Linux 5.19.3

### Environment type (Please select at least one enviroment where you face this issue)

- [X] Self-Hosted
- [ ] Microsoft Hosted
- [ ] VMSS Pool
- [ ] Container

### Azure DevOps Server type

Azure DevOps Server (Please specify exact version in the textbox below)

### Azure DevOps Server Version (if applicable)

Azure DevOps Server 2022 version 19.205.33122.1

### Operation system

Arch Linux 5.19.3

### Version controll system

Azure Repos Git

### Relevant log output

```shell
[2023-04-17 06:10:02Z ERR  StepsRunner] Caught exception from step: System.Exception: You need to provide at least one not-nullable parameter
   at Agent.Sdk.SystemVersion..ctor(String name, String version) in /home/vsts/work/1/s/src/Agent.Sdk/Util/PlatformUtil.cs:line 435
   at Agent.Sdk.PlatformUtil.GetSystemVersion() in /home/vsts/work/1/s/src/Agent.Sdk/Util/PlatformUtil.cs:line 106
   at Microsoft.VisualStudio.Services.Agent.Worker.TaskRunner.PublishTelemetry(Definition taskDefinition, HandlerData handlerData) in /home/vsts/work/1/s/src/Agent.Worker/TaskRunner.cs:line 582
   at Microsoft.VisualStudio.Services.Agent.Worker.TaskRunner.RunAsync() in /home/vsts/work/1/s/src/Agent.Worker/TaskRunner.cs:line 121
   at Microsoft.VisualStudio.Services.Agent.Worker.StepsRunner.RunStepAsync(IStep step, CancellationToken jobCancellationToken) in /home/vsts/work/1/s/src/Agent.Worker/StepsRunner.cs:line 224
```

**Comment by vmapetr** [comment]
Hi, @joperator thanks for reporting! 
We are working on more prioritized issues at the moment but will get back to this one soon.

**Comment by joperator** [comment]
I've seen that there were two [pre-releases](https://github.com/microsoft/azure-pipelines-agent/releases) last month, but are there any updates on this issue, @vmapetr?

**Comment by SudoNova** [comment]
Hi
Apparently I have the same issue (agent is running on Arch Linux).
OP's workaround worked for me
Kernel=6.4.8

**Comment by DmitriiBobreshev** [comment]
Hi @joperator, the fix was applied in [v3.234.0](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v3.234.0) of the agent, could you please check?

**Comment by joperator** [comment]
I can confirm that this has been fixed in the specified version. Thank you very much.
I'd guess that #4323 can then be closed as well, since you've found another working solution.
